javav2/example_code/autoscale/Readme.md

```diff
@@ -0,0 +1,64 @@
+# Amazon EC2 Auto Scaling code examples for the SDK for Java
+
+## Overview
+This README discusses how to run and test the AWS SDK for Java (V2) examples for Amazon EC2 Auto Scaling.
+
+Amazon EC2 Auto Scaling helps you maintain application availability and allows you to automatically add or remove EC2 instances according to conditions.
```

Comment:
With Amazon EC2 Auto Scaling, you can maintain application availability and automatically add or remove EC2 instances according to conditions.

Reply:
fixed